docs/en/observability/slo-privileges.asciidoc

```diff
@@ -35,11 +35,12 @@ To create a role:
 Set the following privileges for the SLO Editor role:
 
 . Under *Index privileges* in the *Elasticsearch* section, add `.slo-observability-*` to the *Indices* field and `read`, `view_index_metadata`, `write`, and `manage` to the *Privileges* field.
+. Under *Cluster privileges* add `manage_transform` as SLO most likely to create the transform respect to your SLO.
 . Click *Add index privilege*.
 . In the *Indices* field, add the indices for which you plan to create SLOs. Then, add `read` and `view_index_metadata` to the *Privileges* field. The following example shows `logs-*`, but you can specify any indices.
 +
 [role="screenshot"]
-image::images/slo-es-priv-editor.png[Cluster and index privileges for SLO Editor role]
```

Comment:
If this image is no longer needed, you should delete it. Alternatively you could rename your new image `slo-es-priv-editor.png` so it replaces the existing image.